### PR TITLE
Feature/router

### PR DESCRIPTION
**Pull request by Ryan-D-Miller**
# Describe Changes

Please include changes made or which issues were fixed.

# Type of Change
- [x] Feature HTML
- [x] Feature JavaScript
- [ ] Feature CSS 
- [ ] Refactor/Fix Bugs

# Please describe the change made

Please give a short description of the changes that were made below
- Fix:

- Feature:
Add router to move through the application

# How has this been tested?

- [x] In Browser
- [ ] In Console

### CODE REVIEW

File: src/components/App/App.js [hunk 1]
```diff
@@ -24,31 +25,22 @@ function App() {
...
+        <Switch>
+          <Route exact
+            path='/'
+            component={Dashboard}
+          />
+          <Route exact
+            path='/wishlist'
+            component={Wishlist}
+          />
+          <Redirect to='/' />
```
Comment (daomeow) [inline src/components/App/App.js:41]: I'll add the Game details to this when we push this to main/on my PR. Thanks Ryan

Comment (daomeow) [inline src/components/App/App.js:41]: WOW this redirect to home worked inside the switch?! Interesting!!!

File: src/components/App/App.js [hunk 2]
```diff
@@ -24,31 +25,22 @@ function App() {
       .catch((error) => console.log(error));
   }, []) 
 
-
-  const games = useSelector(state => state.boardGames)
-
-  const gameImages = games.map(game => {
```
Comment (errabun) [inline src/components/App/App.js:30]: Nice to see this logic taking place outside of App now!

**Comment by daomeow** [comment]
Crushed this!

File: src/components/Dashboard/Dashboard.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
+import './Dashboard.css';
+import React, { useEffect } from 'react'
+import { getGames } from '../../apiCalls.js';
+import { useSelector, useDispatch } from 'react-redux';
+import { addGames, addWish } from '../../actions';
+import {Route, Link, Switch, Redirect } from 'react-router-dom';
```
Comment (errabun) [inline src/components/Dashboard/Dashboard.js:6]: Looks like we are just using Link in this component, we probably don't need to import all these methods here

**Review by errabun — APPROVED** [review]

**Pull request merged**